dask_cuda/benchmarks/common.py

```diff
@@ -155,6 +155,12 @@ def run(client: Client, args: Namespace, config: Config):
             p2p_bw,
         )
 
+    if args.gather_shuffle_stats:
+        from rapidsmpf.integrations.dask.shuffler import gather_shuffle_statistics
+
+        shuffle_stats = gather_shuffle_statistics(client)
+        print(shuffle_stats, flush=True)
+
```

Comment:
Let's move this into the `pretty_print_results` implementation in `local_cudf_shuffle.py`.